modules/tectonic/resources/tectonic.sh

```diff
@@ -96,22 +93,8 @@ wait_for_pods() {
   set -e
 }
 
-asset_cleanup() {
-  echo "Cleaning up installation assets"
-
-  # shellcheck disable=SC2034
-  for d in "manifests" "auth" "tectonic" "tls"; do
-      rm -rf "$${ASSETS_PATH:?}/$${d:?}/"*
-  done
-
-  # shellcheck disable=SC2034
-  for f in "bootkube.sh" "tectonic.sh" "tectonic-wrapper.sh"; do
-      rm -f "$${ASSETS_PATH:?}/$${f:?}"
-  done
-}
-
 # chdir into the assets path directory
-cd "$ASSETS_PATH/tectonic"
+cd "tectonic"
```

Comment:
Can we update the service file to launch this script in the asset directory?  Then we could drop this `cd` entirely.